VideoLocker/src/test/java/org/edx/mobile/util/VersionTest.java

```diff
@@ -0,0 +1,143 @@
+package org.edx.mobile.util;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+/**
+ * Tests for verifying implementation correctness of {@link Version}.
+ */
+public class VersionTest {
+    /**
+     * Verify that a new instance created with a two-dot release
+     * string (x.x.x) is parsed correctly, setting the major,
+     * minor, and patch version properties.
+     */
+    @Test
+    public void newInstance_withTwoDotRelease_isParsedCorrectly() {
```

Comment:
Actually, I would say that it does; Since there is only 1 single guideline on that page: `Follow test method naming conventions and use an underscore to separate what is being tested from the specific case being tested. This style makes it easier to see exactly what cases are being tested.`

This method is doing exactly that.